src/workerd/api/kv.h

```diff
@@ -96,6 +97,17 @@ class KvNamespace: public jsg::Object {
     JSG_METHOD_NAMED(delete, delete_);
   }
 
+protected:
+  kj::Own<kj::HttpClient> getHttpClient(
+      IoContext& context,
+      kj::HttpHeaders& headers,
+      kj::OneOf<LimitEnforcer::KvOpType, kj::StringPtr> opTypeOrName,
+      kj::String urlStr
```

Comment:
Seems like this could be `kj::StringPtr` and all of the calls could drop the `kj::str` invocations.